src/main/scala/codacy/swiftlint/SwiftLint.scala

```diff
@@ -59,13 +59,18 @@ object SwiftLint extends Tool {
           Some(writeConfigFile(patternsToLint).toString)
       }
 
+      // Remove native configuration file if configuration is defined
+      // so that swiftlint does not use it when running
+      if (config.isDefined) {
```

Comment:
Good choice to run side effects here 👏instead of in pattern matching

Reply:
Yeah, it gets better 👌